PrestaShopCorp/psxmarketingwithgoogle
# product filter count

US => [EMKTG-3291](https://forge.prestashop.com/browse/EMKTG-3291)
To test :
You can check the debounce with network open (0.5s after change filter you can see the call).
For testing long loading message you can pass offline in network.
If you want to test the method to stop the current call (long time respond) and relaunch one need to be on slow 3g with big shop (like 10k product).
